src/MIDebugEngine/AD7.Impl/AD7Module.cs

```diff
@@ -71,6 +73,17 @@ int IDebugModule2.GetInfo(enum_MODULE_INFO_FIELDS dwFields, MODULE_INFO[] infoAr
                     info.m_dwLoadOrder = this.DebuggedModule.GetLoadOrder();
                     info.dwValidFields |= enum_MODULE_INFO_FIELDS.MIF_LOADORDER;
                 }
+                if ((dwFields & enum_MODULE_INFO_FIELDS.MIF_TIMESTAMP) != 0)
+                {   
+                    long ft = File.GetLastWriteTimeUtc(this.DebuggedModule.SymbolPath).ToFileTime();
```

Comment:
>this.DebuggedModule.SymbolPath [](start = 55, length = 30)

I think you are missing a null check against `this.DebuggedModule.SymbolPath`

Reply:
Actually, it seems like this should probably be looking at `Name` rather than SymbolPath.

---
In reply to: [501252420](https://github.com/microsoft/MIEngine/pull/1054#discussion_r501252420) [](ancestors = 501252420)